src/azureExtensions.ts

```diff
@@ -88,6 +88,14 @@ export const azureExtensions: IAzExtMetadata[] = [
             'microsoft.dbforpostgresql/flexibleservers',
         ],
         reportIssueCommandId: 'azureDatabases.reportIssue'
+    },
+    {
+        name: 'vscode-azurecontainerapps',
+        label: 'Container Apps',
+        resourceTypes: [
+            'microsoft.app/containerapps'
+        ],
+        reportIssueCommandId: 'containerApps.reportIssue'
     }
```

Comment:
This was included in https://github.com/microsoft/vscode-azureresourcegroups/pull/157, do you need to merge origin/unified?